docs/tags.md

```diff
@@ -0,0 +1,18 @@
+---
+id: tags
+title: Tagging Features
+---
+
+> This feature was introduced in Unleash vx.y.z
+
+Do you want to filter your features to avoid having to see all features belonging to other teams than yours? Do you want to write a plugin that only gets notified about changes to features that your plugin knows how to handle?
```

Comment:
```suggestion
Do you want to filter your features to avoid having to see all features belonging to other teams than your own? Do you want to write a plugin that only gets notified about changes to features that your plugin knows how to handle?
```